
From: "Luiz Fernando N. Capitulino" <lcapitulino@prefeitura.sp.gov.br>

drivers/net/at1700.c does not compile without CONFIG_MCA_LEGACY set.

As CONFIG_MCA_LEGACY depends on CONFIG_MCA, we can use only
CONFIG_MCA_LEGACY, insteed of "MCA && MCA_LEGACY".

Signed-off-by: Luiz Capitulino <lcapitulino@prefeitura.sp.gov.br>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/net/Kconfig~net-at1700c-depends-on-mca_legacy drivers/net/Kconfig
--- 25/drivers/net/Kconfig~net-at1700c-depends-on-mca_legacy	2004-06-21 23:17:52.137443776 -0700
+++ 25-akpm/drivers/net/Kconfig	2004-06-21 23:17:52.152441496 -0700
@@ -883,7 +883,7 @@ source "drivers/net/tulip/Kconfig"
 
 config AT1700
 	tristate "AT1700/1720/RE1000Plus(C-Bus) support (EXPERIMENTAL)"
-	depends on NET_ETHERNET && (ISA || MCA) && EXPERIMENTAL
+	depends on NET_ETHERNET && (ISA || MCA_LEGACY) && EXPERIMENTAL
 	select CRC32
 	---help---
 	  If you have a network (Ethernet) card of this type, say Y and read
_
